Source/Scene/ImplicitTileset.js

```diff
@@ -64,13 +64,21 @@ export default function ImplicitTileset(
    */
   this.metadataSchema = metadataSchema;
 
+  const boundingVolume = tileJson.boundingVolume;
+  if (hasExtension(boundingVolume, "3DTILES_bounding_volume_S2")) {
+    // Merge the extension with the boundingVolume for consistency with 3D Tiles 1.1
+    boundingVolume.s2Cell =
+      boundingVolume.extensions["3DTILES_bounding_volume_S2"];
+    delete boundingVolume.extensions;
```

Comment:
Careful, I think it's okay to `delete extensions["3DTILES_bounding_volume_S2"]` here, but I wouldn't delete the extensions completely just to keep things future-proof.